### PR TITLE
Update to game protocol 12.91

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
* Update protocol for login in version 12.91.
* Although the code has been updated to `uint16_t` , purchases are still limited to 100 items.  The change was necessary in this pull request to fix crash when opening the NPC window.
* I will not make/bring the code that brings this increase in quantity, as it is complex and a little big, I will focus only on the protocol part.

**Thanks:**
Zbizu [Zbizu@users.noreply.github.com](mailto:Zbizu@users.noreply.github.com)
Marcos [66353315+marcosvf132@users.noreply.github.com](mailto:66353315+marcosvf132@users.noreply.github.com)

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/protocolgame.cpp [hunk 1]
```diff
@@ -2039,7 +2039,6 @@ void ProtocolGame::sendSaleItemList(const std::list<ShopInfo>& shop)
 
 	NetworkMessage msg;
 	msg.addByte(0x7B);
-	msg.add<uint64_t>(playerBank + playerMoney); // deprecated and ignored by QT client. OTClient still uses it.
```
Comment (EPuncker) [inline src/protocolgame.cpp:2042]: why was this removed?

Comment (omarcopires) [reply src/protocolgame.cpp:2042]: I'm not sure about its use, I don't know if it's still used by otclient.

Comment (ranisalt) [reply src/protocolgame.cpp:2042]: OTClient has a system to handle multiple protocols, they can skip this field for specific versions, it's OK to change

Comment (Zbizu) [reply src/protocolgame.cpp:2042]: Mehah's OTC no longer use that line, it can be removed safely

File: src/networkmessage.cpp [hunk 2]
```diff
@@ -95,6 +95,12 @@ void NetworkMessage::addItem(uint16_t id, uint8_t count)
 		addByte(0x00); // quiver ammo count
 	} else if (it.classification > 0) {
 		addByte(0x00); // item tier (0-10)
+	} else if (it.showClientCharges) {
+		add<uint32_t>(it.charges);
+		addByte(0x00); // ??
```
Comment (Zbizu) [inline src/networkmessage.cpp:100]: boolean "isBrandNew"

you can hide charges/duration for brand-new items in client
